sdk/tests/client/addresses.rs

```diff
@@ -173,7 +170,10 @@ async fn address_generation() {
             .with_coin_type(address.coin_type)
             .with_account_index(address.account_index)
             .with_range(address.address_index..address.address_index + 1)
-            .with_internal_addresses(address.internal)
+            .with_options(GenerateAddressOptions {
```

Comment:
We could do this using a GenerateAddressOptions ::from(AddressData) or something, but thats not much prettier i guess

Reply:
yea never mind, would only be usefull to clean this test, thats useless then